src/EnergyPlus/DualDuct.cc

```diff
@@ -1529,7 +1529,6 @@ namespace DualDuct {
         this->dd_airterminalOutlet.AirMassFlowRateMaxAvail = MassFlow;
         this->dd_airterminalOutlet.AirMassFlowRateMinAvail = this->ZoneMinAirFrac * this->dd_airterminalHotAirInlet.AirMassFlowRateMax;
         this->dd_airterminalOutlet.AirEnthalpy = Enthalpy;
-        this->OutdoorAirFlowRate = MassFlow * AirLoopOAFrac;
```

Comment:
this is moved to a new common member function CalcOutdoorIAirVolumeFlowRate() added for AirTerminalDualDuct:VAV and AirTerminalDualDuct:ConstantVolume objects.